examples/with_refresh_token.rs

```diff
@@ -95,7 +86,7 @@ async fn main() {
     println!(">>> Session three, running some requests:");
     let spotify = AuthCodeSpotify::new(creds.clone(), oauth.clone());
     spotify
-        .refresh_token(&refresh_token)
```

Comment:
Oooh, it turns out that it's impossible to `refresh_token` without the `token.refresh_token`.

When we try to `refresh_token` by `refetch_token` function without `token.refresh_token`, it will simply `panicked at 'Rspotify not authenticated', rspotify/src/clients/base.rs:104:55`, because the `auth_headers` function need an authenticated token.